autopush/db.py

```diff
@@ -468,6 +468,10 @@ def __init__(self, **kwargs):
         conf = kwargs
         if not conf.get("endpoint_url"):
             conf["endpoint_url"] = os.getenv("AWS_LOCAL_DYNAMODB")
+        creds = os.getenv("AWS_LOCAL_CREDS")
```

Comment:
autopush-rs hardcodes creds to bogus whenever AWS_LOCAL_DYNAMODB is set. I'm thinking we do the same here (killing AWS_LOCAL_CREDS), as I don't think anyone will ever need custom creds when pointing to a local ddb

Reply:
I was debating that myself. The only potential reason I could come up with to keep them separate was the low percentile chance that they actually had some specific credentials that they wanted to use. 

Yeah, I have really dumb arguments with myself.